src/index.html

```diff
@@ -0,0 +1,123 @@
+                    <button class="slider-countries-cases__right"></button>                                              
+                </div>
+            </div>           
+            <div class="map">
+                <div id = "map" style = "width:100%; height:100%;"></div>
+            </div>
+            <div class="details-wrapper">              
+                <div class="global-cases">
+                    <span class="global-cases__title">Total Covid-19 cases</span>                        
+                    <div class="global-cases__volume"></div>
+                    <div class="slider-global-cases">                        
+                        <button class="slider-global-cases__left"></button>                                               
+                        <span class="slider-global-cases__content">Confirmed Cases</span>                        
+                        <button class="slider-global-cases__right"></button>                                              
+                    </div>  
+                </div>
+                <div class="chart">
+                    <div id="slider">
+                        <div id="polosa">
+                          <div class="d"><canvas class="canvas" id="myChart" height="250px"></canvas></div> 
+                          <div class="d"><canvas class = "canvas" id="myChartDeath" height="250px"></canvas></div>
+                          <div class="d"><canvas class = "canvas" id="myChartHealed" height="250px"></canvas></div> 
+                        </div>
+                </div>
+                    <div class="legend">
+                       <button id="slider-right">
+                        <img class="arrow__legend" src="https://www.flaticon.com/svg/static/icons/svg/467/467272.svg" alt="">
+                       </button> 
+                        <span>Next chart</span>
+                        <button id="slider-left">
+                            <img class="arrow__legend" src="https://www.flaticon.com/svg/static/icons/svg/467/467280.svg" alt="">
+                        </button>
+                    </div>
+                    <div class="d"><canvas class="canvas" id="chartBar" height="250px"></canvas></div> 
+                </div>
+            </div>            
+        </div>       
+        <div class="footer-wrapper">
+            <div class="rs-logo">
+                <img src="https://rs.school/images/rs_school_js.svg" alt="RSSchool">
+            </div>
+            <div class="copyright">
+                2020 © Covid-19 Dashboard
+            </div>
+            <div class="author">
+                <a href="https://github.com/denisafa">@denisafa,</a>
+                <a href="https://github.com/alenvoina">@alenvoina,</a>
+                <a href="https://github.com/pobgl">@pobgl</a>                
+            </div>
+        </div>    
+    </div>    
+    <script>
+        document.getElementById('slider-left').onclick = sliderLeft;
```

Comment:
I think you can do the logic for slider in a separate js file. It's not good to embed the code in scrypt tag